### DISCUSSION TITLE
Setting max-width for AppShell

### DISCUSSION THREAD

**Discussion by pang0103** [mantinedev/mantine#6437] [Q&A] [answered]
I want to set a max-width and also keep the AppShell center in the screen so that it is more user friendly for ultra-wide screen.

Below is the result desired:
![image](https://github.com/mantinedev/mantine/assets/32352542/1171d4bd-fad8-4b00-a41e-db0b07c29308)

I tried to set the AppShell style with maxWidth, and margin: auto, but the style applied to <AppShell.Main> instead of the AppShell component.

Thanks!

**Comment by rtivital** [comment, marked as answer]
Not possible. AppShell has fixed position, max-width cannon be applied.
